src/main/java/org/testng/xml/Parser.java

```diff
@@ -23,9 +24,17 @@
 
   /** The URL to the deprecated TestNG DTD. */
   public static final String DEPRECATED_TESTNG_DTD_URL = "http://beust.com/testng/" + TESTNG_DTD;
+  private static final String HTTPS_DEPRECATED_TESTNG_DTD_URL = "https://beust.com/testng/" + TESTNG_DTD;
```

Comment:
You were right in your previous comment. DEPRECATED is not the best term. You can change it by PREVIOUS or OLD if you want.

Reply:
Fixed